app/services/provider_setup.rb

```diff
@@ -28,6 +28,12 @@ def next_relationship_pending
 
   def relationships_pending
     auth = ProviderAuthorisation.new(actor: @provider_user)
-    auth.training_provider_relationships_that_actor_can_manage_organisations_for.select { |prp| prp.setup_at.blank? || prp.invalid? }
+    auth.training_provider_relationships_that_actor_can_manage_organisations_for.select do |prp|
```

Comment:
Can you use a more descriptive variable name. `relationship` I think would be a good fit.